templates/service.yaml

```diff
@@ -16,6 +16,12 @@ spec:
 {{- if (and (eq .Values.service.type "ClusterIP") (not (empty .Values.service.clusterIP))) }}
   clusterIP: {{ .Values.service.clusterIP }}
 {{- end }}
+{{- if (not (empty .Values.service.externalIPs)) }}
+  externalIPs:
+    {{- range .Values.service.externalIPs }}
+    - {{ . | quote }}
+    {{- end }}
+{{- end }}
```

Comment:
This block (L19-L24) could be simplified using a combination of helpers (`toYaml` and `nindent`) and collapsing the if-conditional expression (`if not empty X` can be reduced to `if X`):

```yaml
{{- if .Values.service.externalIPs }}
  externalIPs: {{ toYaml .Values.service.externalIPs | nindent 4 }}
{{- end }}
```